app/uk/gov/hmrc/helptosave/connectors/FrontendConnector.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosave.connectors
+
+import com.google.inject.{ImplementedBy, Inject, Singleton}
+import uk.gov.hmrc.helptosave.config.WSHttp
+import uk.gov.hmrc.helptosave.models.NSIUserInfo
+import uk.gov.hmrc.helptosave.util.Logging
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
+import uk.gov.hmrc.play.config.ServicesConfig
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@ImplementedBy(classOf[FrontendConnectorImpl])
+trait FrontendConnector {
+
+  def createAccount(userInfo: NSIUserInfo)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse]
+}
+
+@Singleton
+class FrontendConnectorImpl @Inject() (http: WSHttp)(implicit hc: HeaderCarrier)
+  extends FrontendConnector with ServicesConfig with Logging {
+
+  val createAccountURL: String = getString("microservice.services.help-to-save-frontend.url")
+
+  override def createAccount(userInfo: NSIUserInfo)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = {
+    http.post(createAccountURL, userInfo)
```

Comment:
do we want to do a `recover` block here to better control the error response in case the future fails?